### ISSUE TITLE
It does not add an empty line between groups

### ISSUE THREAD

**Issue by maksimsemenov** [hudochenkov/stylelint-order#133]
I'm trying to use `order/properties-order` rule with `emptyLineBefore: 'always'`, but it does not add an empty line.

Here is the repo with a reproducible example: https://github.com/maksimsemenov/stylelint-order-bug

If you run `npx stylelint src/**/*` it show an errors, that empty lin is missing. But if you run it with the fix options (`npx stylelint --fix src/**/*`) it doesn't show any errors, but empty lines are not added as well.

I was trying to debug it, and it seems that `addEmptyLineBefore` function is called and it changes the node, but these changes are not written to the file. However, the properties for reordered, so it writes some changes to the files, but not the empty line.

**Comment by hudochenkov** [comment]
Thanks for providing reproducible example!

`declaration-empty-line-before`, which is [set in `stylelint-config-standard`](https://github.com/stylelint/stylelint-config-standard/blob/8ea27ab960ee9eaf02ae6c689675f62c2a3ab3de/index.js#L50) removes an empty line. You need to disable this rule.

**Comment by maksimsemenov** [comment]
Ohh, thanks! That makes sense now. Thanks for explaining this.

Will it be helpful to add this into docs? I can work on PR, but just want to check if it is something that is needed

**Comment by hudochenkov** [comment]
Sounds like a good idea.

**Comment by RicLeP** [comment]
What value is needed for `declaration-empty-line-before`? I’ve tried lots of values but with no success.

**Comment by thasmo** [comment]
I want to use ...

```
        'declaration-empty-line-before': ['always', {
            except: ['first-nested', 'after-comment', 'after-declaration'],
        }],
```

... to force an empty line before a block of declarations.

Unfortunately setting this, empty lines between declarations via `order/properties-order` do not work anymore.

**Comment by Astrael1** [comment]
I see that reference to [stylelint-config-standard](https://github.com/stylelint/stylelint-config-standard/blob/8ea27ab960ee9eaf02ae6c689675f62c2a3ab3de/index.js#L50) is already present in the docs for [properties-order](https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-order/README.md)

@hudochenkov could you close this issue? Or is there anything more to do?
